content/generic.rst

```diff
@@ -21,6 +21,9 @@ something is wrong.
 
 * Rule #3: If it ain't broke, don't fix it!
 
+* Ammendment to Rule #3: If it's not totally broken but still wastes a lot of your time, fix it! 
+  This means all the FIXME tags should be a priority when 'refactoring <http://ronjeffries.com/xprog/articles/refactoring-not-on-the-backlog/>'.
```

Comment:
FIXME tags are just in code markers for tech debt... my understanding from that article is that tech dept should have the lowest priority and you should never fix a tech debt just so ... but only work on it if you will hit it while working on your main high priority task